Formula/k/kubefirst.rb

```diff
@@ -1,8 +1,8 @@
 class Kubefirst < Formula
   desc "GitOps Infrastructure & Application Delivery Platform for kubernetes"
   homepage "https://kubefirst.io/"
-  url "https://github.com/kubefirst/kubefirst/archive/refs/tags/v2.2.17.tar.gz"
-  sha256 "cb0d4e265b7bc2c75c2a5898dde3af126a2bc159b203cd61901611149f8d7433"
+  url "https://github.com/kubefirst/kubefirst/archive/refs/tags/v2.3.3.tar.gz"
+  sha256 "9151109dfa8b147d11f33b474a8e40f6768a409849d48be6b7cf8ef1b7df6542"
```

Comment:
```suggestion
  sha256 "c217488a7a3ae5aae426ef101bf63535d44e8b0950901c7cb2708d91800fdc94"
```

Reply:
yeah, I am getting the same as CI

```
$ curl -sL https://github.com/kubefirst/kubefirst/archive/refs/tags/v2.3.3.tar.gz | shasum -a 256
c217488a7a3ae5aae426ef101bf63535d44e8b0950901c7cb2708d91800fdc94  -
```

Reply:
thank you. Committed that suggestion and noted how to properly pull the sha 256 for a github release. 
![image](https://github.com/Homebrew/homebrew-core/assets/6446939/158c631b-3b0e-46c7-bb24-3296938bb0d6)